services/docker/docker-size.service.js

```diff
@@ -4,13 +4,13 @@ const Joi = require('@hapi/joi')
 const prettyBytes = require('pretty-bytes')
 const { nonNegativeInteger } = require('../validators')
 const { latest } = require('../version')
+const { NotFound } = require('..')
+const { BaseJsonService } = require('..')
```

Comment:
No change needed here, but may be worth combining these into a single import 

```suggestion
const { BaseJsonService, NotFound } = require('..')
```